src/main/java/io/appium/java_client/android/AndroidDriver.java

```diff
@@ -208,7 +208,9 @@ public AndroidBatteryInfo getBatteryInfo() {
      */
     public Capabilities getCapabilities() {
```

Comment:
Can you please add Nullable annotation to methods which can return null? It seems like we are not going to get any response from Selenium contributors, so will merge this fix as is.